soci/fbs/ztoc.fbs

```diff
@@ -0,0 +1,47 @@
+namespace ztoc;
+
+table Xattr {
+	key : string;
+	value : string;
+}
+
+table FileMetadata {
```

Comment:
- Are they all signed? https://google.github.io/flatbuffers/flatbuffers_guide_writing_schema.html

Reply:
Yep. I'm duplicating what we already have for Ztoc struct. `FileSize` and `SpanId` are signed integers and they translate to `int64` and `int32` respectively.